LoopKitUI/View Controllers/ChartsTableViewController.swift

```diff
@@ -6,17 +6,32 @@
 //
 
 import UIKit
+import Combine
 import HealthKit
 
 /// Abstract class providing boilerplate setup for chart-based table view controllers
 open class ChartsTableViewController: UITableViewController, UIGestureRecognizerDelegate {
 
-    public var preferredGlucoseUnit: HKUnit?
+    public var displayGlucoseUnitObservable: DisplayGlucoseUnitObservable? {
+        didSet {
+            guard let displayGlucoseUnitObservable = displayGlucoseUnitObservable else { return }
+
+            cancellable = displayGlucoseUnitObservable.updatePublisher.sink { [weak self] in
```

Comment:
[nit] Consider below comment, plus:
```
$displayGlucoseUnitObservable
    .sink { [weak self] in self?.unitPreferencesDidChange(to: displayGlucoseUnitObservable.displayGlucoseUnit) }
    .store(in: &cancellables)
```
Not sure if this will work here.